CMakeLists.txt

```diff
@@ -2,6 +2,10 @@ cmake_minimum_required(VERSION 2.8)
 
 project(synfig-studio)
 
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/output/lib)
```

Comment:
As we planning to use that path further i suggest to extract that path `${CMAKE_BINARY_DIR}/output/` into a separate variable `SYNFIG_BUILD_ROOT `

Reply:
That makes sense. Would make things easier to maintain in the future.